Realm/RLMUtil.mm

```diff
@@ -390,6 +391,25 @@ BOOL RLMIsRunningInPlayground() {
     return [[NSBundle mainBundle].bundleIdentifier hasPrefix:@"com.apple.dt.playground."];
 }
 
+std::map<size_t, id(^)(realm::util::Any const&)> cppToObjC_Mappings = {
```

Comment:
This is a very strange way to switch on the type which is verbose, slow, and not guaranteed to work (both types could legally have the same hash code).

The one place that is using this should probably actually be using `realm::Mixed`. `Any` is only appropriate when the value actually could be literally any type, not when it could be one of a fixed set of supported types.

Reply:
In retrospect, this will not work any way. It seems we should not expose `customData` until we are able to handle xjson– none of these types will map properly until then. Going to remove this for now.